TodoList/src/app/screens/Checkbox/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { View, StyleSheet, TouchableOpacity } from 'react-native'
+import { styles } from './styles.js'
+
+export default class Checkbox extends Component {
```

Comment:
`Checkbox` should be a function. It doesn't have internal state and doesn't rely on lifecycle functions